crates/signer/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "eigen-signer"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+repository.workspace = true
+
+[dependencies]
+alloy-consensus.workspace = true
+alloy-network.workspace = true
+alloy-primitives.workspace = true
+alloy-signer.workspace = true
+alloy-signer-aws.workspace = true
+alloy-signer-local.workspace = true
+alloy-rpc-client.workspace = true
+aws-config.workspace = true
+aws-sdk-kms.workspace = true
+eth-keystore.workspace = true
+hex-literal.workspace = true
+thiserror.workspace = true
+tokio.workspace = true
+serde.workspace = true
+url.workspace = true
+testcontainers.workspace = true
```

Comment:
this crate should be placed inside dev-dependencies section also

Reply:
Done